charts/opentelemetry-demo/values.yaml

```diff
@@ -165,18 +165,34 @@ components:
     enabled: true
     servicePort: 8080
     env:
-    - name: OTEL_PYTHON_LOG_CORRELATION
-      value: 'true'
-    - name: PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION
-      value: python
+      - name: OTEL_PYTHON_LOG_CORRELATION
+        value: "true"
+      - name: PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION
+        value: python
     podAnnotations: {}
     #  instrumentation.opentelemetry.io/inject-python: "true"
 
   shippingService:
     enabled: true
     servicePort: 8080
     env:
-    - name: PORT
-      value: '8080'
+      - name: PORT
+        value: "8080"
     podAnnotations: {}
     #  instrumentation.opentelemetry.io/inject-sdk: "true"
+
+opentelemetry-collector:
+  nameOverride: otelcol
+  mode: deployment
+  config:
+    exporters:
+      jaeger:
+        endpoint: "${JAEGER_ADDR}"
+        tls:
+          insecure: true
+    service:
+      pipelines:
+        traces:
+          exporters:
+            - logging
+            - jaeger
```

Comment:
This looks pretty good to me

Reply:
Yes, this is simple enough for me as well.  Still don't love that the collector is doing more than the demo requires, but maybe the subchart is worth that complexity.